backends-common/postgres/src/main/java/org/apache/james/backends/postgres/PostgresConfiguration.java

```diff
@@ -59,6 +59,8 @@ public class PostgresConfiguration {
     public static final String SSL_MODE_DEFAULT_VALUE = "allow";
     public static final String JOOQ_REACTIVE_TIMEOUT = "jooq.reactive.timeout";
     public static final Duration JOOQ_REACTIVE_TIMEOUT_DEFAULT_VALUE = Duration.ofSeconds(10);
+    public static final String ATTACHMENT_STORAGE_ENABLED = "attachment.storage.enabled";
+    public static final boolean ATTACHMENT_STORAGE_ENABLED_DEFAULT_VALUE = false;
```

Comment:
For JMAP users this makes sens to have this `true`

Maybe we can keep the existing behaviour given we provide an easy option to opt out?